app/index.js

```diff
@@ -7,7 +7,7 @@
 
 var path = require('path');
 var SG = require('strong-globalize');
-SG.SetRootDir(path.join(__dirname, '..'));
+SG.SetRootDir(path.join(__dirname, '..'), {autonomousMsgLoading: 'all'});
```

Comment:
I might miss something here, when do we need to set `autonomousMsgLoading` as `all`?